BLE_GattServer_ExperimentalServices/README.md

```diff
@@ -0,0 +1,56 @@
+# Link Loss Service
+
+## Introduction
+This application sends an alert message over a serial terminal once the connection to the client is lost without prior warning.
+The client sets the alert message string by writing a valid byte to a R/W GATT Characteristic inside the Link Loss Service (LLS):
+
+* 0x00 ("No Alert")
+* 0x01 ("Mild Alert") 
+* 0x02 ("High Alert")
+
+The alert ends after a user-defined timeout (= 1 minute).
+
+## Usage
+
+### Hardware Requirements
+Please refer to the main [README](https://github.com/ARMmbed/mbed-os-example-ble/blob/master/README.md).
+
+### Build Instructions
+Please refer to the main [README](https://github.com/ARMmbed/mbed-os-example-ble/blob/master/README.md).
+
+### Demostration
+**Note:** The steps given below demostrate the application using an Android smartphone running nRF Connect 4.24.3. 
+The illustations will be different for other clients.         
+1. Build the application and flash the board
+2. Open a serial terminal on your host to receive serial prints from the application:
```

Comment:
apparently our system is broken and cannot accept numbers other than 1. so lists have to be like this:
1.
1.
1.

The list will still display correctly as makrdown ignores numbers

Someone please correct me if our systems can now deal with numbers

Reply:
Markdown uses `1.` for every item in a numbered list this is by design so if you cut/paste element of a list you don't have to number everything again. It works with numbered list within a numbered list too.